static/src/js/components/Tour/TourSteps.jsx

```diff
@@ -103,9 +107,9 @@ const TourSteps = ({
           preferences.
         </p>
         <p className="search-tour__note">
-          If you want to skip the tour for now, it is always available by clicking
+          If you want to skip the tour for now, it is always available by clicking the
           {' '}
-          <strong>Start Tour</strong>
+          <strong>?</strong>
```

Comment:
Should this be an icon and not just a character?

Reply:
Do you mean something like 
![image](https://github.com/user-attachments/assets/cca58b27-fbfc-4135-8208-139eedf9bcf1)
I think that does look nicer and then its the exact same character

Reply:
I updated this and fixed the line spacing between the `at` added screenshot to PR